src/components/graphs/bargraph/BarGraph.js

```diff
@@ -1,15 +1,57 @@
 import React, { useState, useEffect } from 'react';
-import { Bar } from 'react-chartjs-2';
-import { createDataSet } from '../assets/bargraphAssets';
+import { Bar, HorizontalBar } from 'react-chartjs-2';
+
+import './BarGraph.less';
+
+const graphOptions = {
+  responsive: true,
+  maintainAspectRatio: false,
+  legend: {
+    display: false,
+    position: 'left',
+    labels: {
+      boxWidth: 0,
+    },
+  },
+};
+
+const horizOptions = {
+  ...graphOptions,
+  scales: {
+    xAxes: [
+      {
+        position: 'top',
+        scaleLabel: {
+          fontSize: 14,
+          display: true,
+          labelString: 'Number of Incidents',
+        },
+      },
+    ],
+    yAxes: [
+      {
+        position: 'left',
+        scaleLabel: {
+          fontSize: 14,
+          display: true,
+          labelString: 'U.S. State',
+        },
+      },
+    ],
+  },
+  legend: {
+    ...graphOptions.legend,
+    position: 'top',
+  },
+};
 
```

Comment:
Adding this responsiveness is imperative for the the graphs visualization, great job